tests/test_cli.py

```diff
@@ -0,0 +1,154 @@
+"""
+rna_cd
+Copyright (C) 2018-2019  Leiden University Medical Center, Sander Bollen
+
+This program is free software: you can redistribute it and/or modify
```

Comment:
So this thing I said at the first license notice I did not at that at every license notice. But it applies everywhere.